packages/lumx-react/src/components/slideshow/Slideshow.tsx

```diff
@@ -182,20 +182,69 @@ export const Slideshow: Comp<SlideshowProps, HTMLDivElement> = forwardRef((props
         onChange(currentIndex);
     }, [currentIndex, onChange]);
 
+    const slideshowSlidesId = 'slideshow-slides';
+    const slideshowId = 'slideshow';
```

Comment:
This will actually be a problem when there is multiple slideshow on the same page. 
I think we should make these customizable via props so that we can use the slideshow controls separated from the slideshow but still connected via `aria-controls`
I would propose:
```js
const { id, slidesId } = props;

const slideshowId = useMemo(() => id || uniqId('slideshow'), [id]) 
const slideshowSlidesId = useMemo(() => slidesId || uniqId('slideshow-slides'), [slidesId]) 
```